src/patternfly/components/NotificationBadge/notification-badge.scss

```diff
@@ -58,10 +64,14 @@
   & > i {
     width: var(--pf-c-notification-badge__i--Width);
     height: var(--pf-c-notification-badge__i--Height);
+    line-height: var(--pf-global--LineHeight--md);
```

Comment:
This should be a var, like `--pf-c-notification-badge__i--LineHeight`

Reply:
Also, should this be icon instead of i because we have a system to write out elements and then maybe a follow-up issue for the other two variables in a breaking change